python/lsst/cp/pipe/linearity.py

```diff
@@ -463,13 +525,18 @@ def run(self, inputPtc, dummy, camera, inputDims,
                 # to allow for different linearity coefficients with different
                 # photodiode settings.  The minimization is a least-squares
                 # fit with the residual of
-                # Sum[(S(mu_i) + mu_i)/(k_j * D_i) - 1]**2, where S(mu_i) is
-                # an Akima Spline function of mu_i, the observed flat-pair
+                # Sum[(S(mu_i) + mu_i - O)/(k_j * D_i) - 1]**2, where S(mu_i)
+                # is an Akima Spline function of mu_i, the observed flat-pair
                 # mean; D_j is the photo-diode measurement corresponding to
                 # that flat-pair; and k_j is a constant of proportionality
                 # which is over index j as it is allowed to
                 # be different based on different photodiode settings (e.g.
-                # CCOBCURR).
+                # CCOBCURR); and O is a constant offset to allow for light
```

Comment:
Is this `O` a single value for all pairs?  I'll find out when I get to the fitting code, but I'm just checking if this needs any index as well.

Reply:
And I agree now that this is `O`, with no indexing.